components/form.js

```diff
@@ -71,6 +71,19 @@ export default class Form extends React.Component {
       });
     }
   }
+  isOnlineEvent = e => {
+    let formDataArray = JSON.parse(
+      localStorage.getItem("Failed Form Submission Data")
+    );
+    if (!formDataArray) {
+      return;
+    } else {
```

Comment:
you can avoid else here, just do the IF , return, and continue

so like this

if (!formDataArray) {
       return;
}

       formDataArray.map(formData => {
         postFormData(formData, this.state.token.access_token);
       });
       localStorage.removeItem("Failed Form Submission Data");

Reply:
Done